CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.0.1 / 2022-05-10
+
+### ENHANCEMENT
+- Add utils related unit tests  (#251) 
+- Change the original namespace to fluent (#267)
+- Support logstashPrefix to ClusterOutput (#268)
+- Add its shortname for each CRD  (#269)
+
+### BUGFIX
```

Comment:
Actually, all these 3 files should be checked in the master branch first, and then cherry-pick to release-1.0.
Make sure you release like this in the future.
Now you have to cherry-pick this back to master